.bacon.yml

```diff
@@ -61,4 +61,11 @@ test_suites:
       timeout: '60'
       script_name: publish
       criteria: MERGE
+      queue_name: small
+    - name: e2e-saucelabs 
+      script_path: ../okta-auth-js/scripts
+      sort_order: '8'
+      timeout: '60'
```

Comment:
nit: let's start with a shorter timeout, like `20`, so we are not waiting on bacon for some edge cases.